homeassistant/components/go2rtc/__init__.py

```diff
@@ -222,7 +222,9 @@ def on_messages(message: ReceiveMessages) -> None:
             value: WebRTCMessage
             match message:
                 case WebRTCCandidate():
-                    value = HAWebRTCCandidate(RTCIceCandidate(message.candidate))
+                    value = HAWebRTCCandidate(
+                        RTCIceCandidate(message.candidate, sdp_mid="0")
```

Comment:
Is setting sdp_mid to 0 specified somewhere?

Reply:
This was hardcoded in the front end prior to https://github.com/home-assistant/frontend/pull/22667 and I'll be updating that shortly to hardcode it again if neither sdpMid or sdpMLineIndex is received.

Reply:
This now defaults to `sdpMLineIndex=0` in webrtc-models.